.circleci/config.yml

```diff
@@ -409,6 +410,14 @@ commands:
             pip3 install requests
             python3 scripts/trigger-mobile-metrics.py
 
+  setup-aws-credentials:
+    steps:
+      - run:
+          name: Install mbx-ci and obtain credentials
+          command: |
+            curl -Ls https://mapbox-release-engineering.s3.amazonaws.com/mbx-ci/latest/mbx-ci-linux-amd64 > mbx-ci && chmod 755 ./mbx-ci
```

Comment:
https://github.com/mapbox/mapbox-navigation-android/blob/b586340d33f037ad4ebdd4be452718e246e7fc4c/.circleci/config.yml#L128-L129

nit: There is some duplication here if you can think of a elegant way to combine these